src/styles/variables/_colors.scss

```diff
@@ -1,9 +1,109 @@
+// Black
```

Comment:
Do these variable names line up with what's in Figma at the moment? I think the ultimate DX here would be if these variables matched the Figma colour names, so you don't have to look up what we call it in our code base.